website/versioned_docs/version-1.1/tutorials/structured_config/5_schema.md

```diff
@@ -272,3 +281,10 @@ password: drowssap
 - we override the package to `_here_` to ensure that the package of the schema is the same as the package 
   of the config it's validating.
 
+### A Note about composition order
+ By default, Hydra 1.1 appends `_self_` to the end of the Defaults List. 
+This behavior is new in Hydra 1.1 and different from previous Hydra versions. As such Hydra 1.1  issues a warning if `_self_` is not specified **in the primary config**, asking you to add `_self_` and thus indicate the desired composition order.
+To address the warning, append `_self_` to the end of the Defaults List. Note that in some cases you may want to add `_self_` directly after the schema and before other Defaults List elements. 
```

Comment:
```suggestion
To address the warning while maintaining the new behavior, append `_self_` to the end of the Defaults List. Note that in some cases it may instead be desirable to add `_self_` directly after the schema and before other Defaults List elements. 
```